src/main/java/app/controller/ConfigController.java

```diff
@@ -69,6 +78,20 @@ public String getConfig(Model model) throws IOException {
     @PostMapping
```

Comment:
You save that address here in an extra method. Do we really need that?
Because you could just save that address whenever the config file gets saved. You have the priv key there then anyway... No need for an extra post method afaik